mlir/include/mlir/TableGen/Pattern.h

```diff
@@ -279,6 +300,11 @@ class SymbolInfoMap {
     //   the DAG of the operation, `operandIndexOrNumValues` specifies the
     //   operand index, and `variadicSubIndex` must be set to `std::nullopt`.
     //
+    // * Properties not associated with an operation (ex. as arguments to
+    //   native code) have their corresponding PropConstraint stored in the
+    //   `dag` field. (They'll and set `operandIndexOrNumValues` to -1 to
```

Comment:
"They'll and set" feels like a word missing. How about, With `operandIndexOrNumValues` set to -1 to ...

Is there a way to expose the sentinel in the comment here? (I actually couldn't see where a user would need to know the value of this sentinel)